.travis.yml

```diff
@@ -30,7 +28,3 @@ script:
 
 after_success:
   - codecov
-
-branches:
```

Comment:
Why was this removed?

Reply:
It was removed so I could test my feature branch on Travis CI and be sure it was already passing, I didn't want to submit a broken PR to you! 

I can put it restore it if you prefer, although I would recommend leaving it in so others (and you!) can also test their feature branches too.

Reply:
I don't mind WIP PRs, actually. I added this specifically so I wouldn't see two Travis builds on my own PR branches.